operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/KafkaCluster.java

```diff
@@ -14,7 +15,7 @@
         // due to it being a "reserved" word.
         value = "!has(self.listener) || has(self.__namespace__)",
         message = "Property `listener` may not be used when `namespace` is omitted")
-public class KafkaCluster {
+public class KafkaCluster implements Named {
```

Comment:
As far as I can see Named.uniqueNames isn't called on this class, so I don't think it needs to implement Named.

Reply:
I removed the use of `Named` from the locations in the CRD classes. There was an earlier iteration where it was used for validation, but it didn't work as I expected with the API server.